Tholus.Console/Program.cs

```diff
@@ -1,14 +1,71 @@
 using Cocona;
+using Tholus.Console;
 
-internal class Program
+public record InstallParameters(
+    [Argument("install", Description = "Name of the target installation")]
+    string? Name
+) : ICommandParameterSet;
+
+public record ModBagParameters(
+    [Argument("bag", Description = "Name of the target mod bag")]
+    string Name
+) : ICommandParameterSet;
+
+public record ModParameters(
+    [Argument("name", Description = "Name of the mod")]
+    string Name
+) : ICommandParameterSet;
+
+public record ModSourceParameters(
+    [Argument("source", Description = "Source of the mod to be installed")]
+    string Source
```

Comment:
what is `Source` here? i.e. a URL, file path, ...; can the intended options be documented? (or a todo comment left for the implementation)